README.md

```diff
@@ -13,3 +13,58 @@ Add a Markdown file with the translation to the corresponding article folder. Fo
+| Making setInterval Declarative with React Hooks       | Français                                                            |              |
+| React as a UI Runtime                                 | Español • Français                                                  |              |
+| Why Isn’t X a Hook?                                   | Deutsch • Español • Français • Italiano •  <br> Português do Brasil • 简体中文 |    |
+| The “Bug-O” Notation                                  | Français • 한국어                                                     |              |
+| Preparing for a Tech Talk, Part 2: What, Why, and How | Español • Français                                                  |              |
+| The Elements of UI Engineering                        | Español • Français • Polski • 日本語 • <br> 简体中文 • 한국어           | Português do Brasil |
+| Things I Don’t Know as of 2018                        | Español • Français • Português do Brasil • <br> Svenska • తెలుగు • 日本語 • 简体中文 • 繁體中文 • 한국어 | |
+| Preparing for a Tech Talk, Part 1: Motivation         | Español • Français • Português do Brasil • 한국어                     |              |
+| Why Do React Hooks Rely on Call Order?                | Español • Français                                                  |              |
+| Optimized for Change                                  | Bahasa Indonesia • Deutsch • Español • <br> Français • Polski • Português do Brasil • <br> Български • Українська |       |
+| How Does setState Know What to Do?                    | Français • 日本語 • 简体中文                                           |              |
+| My Wishlist for Hot Reloading                         | Español • Français                                                   |              |
+| Why Do React Elements Have a $$typeof Property?       | Français • Português do Brasil • 日本語                               |              |
+| How Does React Tell a Class from a Function?          | Español • Français • Magyar • Slovenčina • <br>  日本語 • 繁體中文     |               |
+| Why Do We Write super(props)? | Español • Français • Italiano • Magyar • <br> Nederlands • Norsk • Português do Brasil • <br> Slovenčina •Tiếng Việt • Türkçe • srpski • <br> Čeština • فارسی • ไทย • မြန်မာဘာသာ • <br> 日本語 • 简体中文 • 繁體中文 |        |
+
+Once your translation is complete, please submit a PR and tag the maintainer(s) of your language for revivew.
+
+| Language     		    |      Maintainer(s)      | 
+|---------------------|-------------------------|
+| English             |  @gaearon               |
+| Русский             |                         |
+| Türkçe     			    |                         |
+| Español    			    |                         |
+| 한국어           	    |                         |
+| Svenska    			    |                         |
+| Italiano        	  |                         |
+| Bahasa Indonesia    |                         |
+| Português do Brasil |                         |
```

Comment:
```suggestion
| Português do Brasil |  @WendellAdriel          |
```

Just adding myself as a maintainer in the table :wink:

Reply:
Ok I am going to add it from my end.

Reply:
This table looks okay in my text editor but looks so messy when I use `git diff` and here...😫 
<img width="419" alt="screen shot 2019-02-11 at 6 04 29 am" src="https://user-images.githubusercontent.com/31831596/52568199-ee835300-2dc2-11e9-8c4c-790ff72bd3bb.png">

Reply:
Thank you so much @chloewlin!!!

Reply:
Yeah, sometimes this happens with me too.
I really appreciate when tables are "organized", but it's hard to make them to be ok in text editor, console and github at the same time 😭

Reply:
Ahh. Glad to know that I am not crazy. 😊